wiring/src/spark_wiring_network.cpp

```diff
@@ -53,6 +53,25 @@ NetworkClass& NetworkClass::from(network_interface_t nif) {
     }
 }
 
+network_interface_t NetworkClass::from(NetworkClass& network) {
```

Comment:
Should we just implement a way to get `iface_`?

Reply:
```    
operator network_interface_t() const {
        return iface_;
    }
```
Is this supposed to be a function call operator or something else?

Reply:
https://en.cppreference.com/w/cpp/language/cast_operator

Reply:
Ah ok, so we already have this: `static_cast<network_interface_t>(network)`, ill remove this function then as its redundant